src/objects/Gift_Entry_Feature_Gate_Settings__c.object

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <customSettingsType>List</customSettingsType>
```

Comment:
What was the reasoning behind using a list custom setting? I think this may be better handled with a hierarchical setting (similar to the the Gift Entry Settings). And actually - should we just add this field to the Gift Entry settings we already introduced as part of this build? @psadie-sfdo